tests/streaming.rs

```diff
@@ -273,27 +272,28 @@ fn it_streams_chunks_without_pausing_until_end_of_stream() {
     module
         .call_proxy_on_request_headers(http_context, 0, false)
         .expect_log(Some(LogLevel::Debug), Some("#2 on_http_request_headers"))
-        .expect_get_header_map_value(Some(MapType::HttpRequestHeaders), Some(":authority"))
-        .returning(Some("cars.toystore.com"))
         .expect_log(
             Some(LogLevel::Debug),
-            Some("#2 action_set selected some-name"),
+            Some("Getting property: `request.host`"),
         )
-        .expect_get_header_map_value(Some(MapType::HttpRequestHeaders), Some("traceparent"))
-        .returning(None)
-        .expect_get_header_map_value(Some(MapType::HttpRequestHeaders), Some("tracestate"))
-        .returning(None)
-        .expect_get_header_map_value(Some(MapType::HttpRequestHeaders), Some("baggage"))
-        .returning(None)
+        .expect_get_property(Some(vec!["request", "host"]))
+        .returning(Some("cars.toystore.com".as_bytes()))
         .expect_log(
             Some(LogLevel::Debug),
-            Some("get_property: path: [\"request\", \"method\"]"),
+            Some("Selected blueprint some-name for hostname: cars.toystore.com"),
         )
-        .expect_get_property(Some(vec!["request", "method"]))
-        .returning(Some(b"POST"))
+        .expect_log(
+            Some(LogLevel::Debug),
+            Some("#2 pipeline built successfully"),
+        )
+        .expect_get_buffer_bytes(Some(BufferType::HttpResponseBody))
+        .failing_with(Status::BadArgument)
+        .expect_get_buffer_bytes(Some(BufferType::HttpResponseBody))
+        .failing_with(Status::BadArgument)
```

Comment:
Confused why this gets called twice by the module, both with for this buffer, but once with `max_size` `0`, then again (from my code) with value `10`...